lib/modules/manager/regex/readme.md

```diff
@@ -130,3 +130,46 @@ The `Dockerfile` is documented better as well.
 
 The syntax in the example is arbitrary and you can set your own syntax.
 If you do, update your `matchStrings` regex!
+
+### Using regexManager to update the dependency name in addition to version
+
+#### Updating `gitlab-ci include` dep names
+
+If the location of files used in gitlab-ci `includes:` has changed you can use the regex manager to update the `depName` as well as the version.
+
+When doing so you may find you need to specify a second `matchString` for the new name, as when the regex manager validates the change it will not always detect that the new value is valid.
```

Comment:
```suggestion
You can use the regex manager to update the `depName` and the version.
This can be handy when the location of files referenced in gitlab-ci `includes:` fields has changed.

You may need to set a second `matchString` for the new name to ensure the regex manager can detect the new value.
For example:
```